cpp/tests/utilities/column_utilities.cu

```diff
@@ -758,9 +757,6 @@ struct column_comparator {
                   size_type fp_ulps,
                   int depth = 0)
   {
-    CUDF_EXPECTS(lhs_row_indices.size() == rhs_row_indices.size(),
-                 "Mismatch in row counts to compare");
-
     // compare properties
```

Comment:
why was the row count comparison removed here?

Reply:
That means the result should be "false" instead of throwing exception.

Reply:
Correct.  It resulted in inconsistent behavior.